crates/uv-python/src/discovery.rs

```diff
@@ -1309,6 +1309,16 @@ impl PythonRequest {
         if let Ok(request) = PythonDownloadRequest::from_str(value) {
             return Self::Key(request);
         }
+        // Check if the value is current active pyenv virtualenv
+        // It mean that name is end of path provided by PYENV_VIRTUAL_ENV variable
+        if let Some(pyenv_virtual_env) = std::env::var_os("PYENV_VIRTUAL_ENV") {
```

Comment:
`pyenv` sets both `PYENV_VIRTUAL_ENV` and `VIRTUAL_ENV` to the same value. Maybe should here `VIRTUAL_ENV`?

Reply:
We already read `VIRTUAL_ENV` — is it not working for you?

Reply:
No. It do not work.